src/CalcManager/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_library(CalculatorEngine
```

Comment:
When #212 lands, you should also add `Ratpack` library.

Reply:
@janisozaur `Ratpack` support is non-trivial due to dependencies on `intsafe.h` and should be added in a separate PR.

Reply:
The C++ standard does not support flexible array members inside structs:

https://github.com/Microsoft/calculator/blob/7a48f66807fe87c98af9781fda5a526faa13aa86/src/CalcManager/Ratpack/ratpak.h#L53-L66

`Ratpack`, however, heavily relies on this nonstandard extension, pointer arithmetic and memory reordering, which makes the port a non-trivial change that should be handled in a separate PR instead.

I have created an experimental version using `std::vector` in https://github.com/fwcd/WindowsCalculator/tree/ratpack that simply exposes the underlying data array. This compiles with clang, although it is far from being an ideal solution.

Reply:
Tracked in https://github.com/fwcd/WindowsCalculator/pull/2.